cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/encryption/DiskEncryptionSetCreationRequest.java

```diff
@@ -0,0 +1,162 @@
+package com.sequenceiq.cloudbreak.cloud.model.encryption;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.sequenceiq.cloudbreak.cloud.CloudPlatformAware;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.Platform;
+import com.sequenceiq.cloudbreak.cloud.model.Region;
+import com.sequenceiq.cloudbreak.cloud.model.Variant;
+
+public class DiskEncryptionSetCreationRequest implements CloudPlatformAware {
```

Comment:
Related to the "DES flow idempotency": A new ID field has to be added. I will update this comment later. TODO exact name & type.